workshops/guided-projects/guided_project_3.ipynb

```diff
@@ -0,0 +1,414 @@
+{
```

Comment:
<p>Suggestion: Consider clarifying this comment a bit more. Are you recommending learners use the covertype dataset again like Guided Project 2? Are you saying use your own dataset but if you don't have one readily available re-use the covertype dataset like in Guided Project 2? Or do you envision the learner flow through these projects that all learners complete #1 and then they could choose to either complete #2 or #3? If this fork pattern is intended, then maybe consider adding (optional) to the heading? Choose #2 OR #3?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>